diff --git a/interop/utils/logging.go b/interop/utils/logging.go
index 3a6940ad..faa96ba8 100644
--- a/interop/utils/logging.go
+++ b/interop/utils/logging.go
@@ -37,7 +37,7 @@ func GetQLOGWriter() (func(perspective logging.Perspective, connID []byte) io.Wr
 		}
 	}
 	return func(_ logging.Perspective, connID []byte) io.WriteCloser {
-		path := fmt.Sprintf("%s/%x.qlog", strings.TrimRight(qlogDir, "/"), connID)
+		path := fmt.Sprintf("%s/%x.sqlog", strings.TrimRight(qlogDir, "/"), connID)
 		f, err := os.Create(path)
 		if err != nil {
 			log.Printf("Failed to create qlog file %s: %s", path, err.Error())
