diff --git a/interop/server/main.go b/interop/server/main.go
index 867044b4..cd021233 100644
--- a/interop/server/main.go
+++ b/interop/server/main.go
@@ -19,13 +19,14 @@ import (
 var tlsConf *tls.Config
 
 func main() {
-	logFile, err := os.Create("/logs/log.txt")
-	if err != nil {
-		fmt.Printf("Could not create log file: %s\n", err.Error())
-		os.Exit(1)
-	}
-	defer logFile.Close()
-	log.SetOutput(logFile)
+	//logFile, err := os.Create("/logs/log.txt")
+	//if err != nil {
+	//	fmt.Printf("Could not create log file: %s\n", err.Error())
+	//	os.Exit(1)
+	//}
+	//defer logFile.Close()
+	//log.SetOutput(logFile)
+	log.SetOutput(os.Stderr)
 
 	keyLog, err := utils.GetSSLKeyLog()
 	if err != nil {
